Source/Core/DolphinQt2/Config/ControllersWindow.cpp

```diff
@@ -93,7 +93,10 @@ void ControllersWindow::CreateGamecubeLayout()
   m_gc_box = new QGroupBox();
   m_gc_layout = new QFormLayout();
   m_gc_label = new QLabel();
-  m_gc_label->setPixmap(QPixmap(m_gamecube_icon));
+
+  auto pixmap = QPixmap(m_gamecube_icon).scaled(QSize(64, 64));
```

Comment:
Just scaling the image is doing to look ugly. You need to load the `@2x` version (or higher) when the user is using a high DPI.